stagemonitor-core/src/main/java/org/stagemonitor/core/CorePlugin.java

```diff
@@ -396,6 +397,33 @@
 			.configurationCategory(CORE_PLUGIN_NAME)
 			.tags("elasticsearch", "advanced")
 			.buildWithDefault(5);
+	private final ConfigurationOption<Collection<String>> remotePropertiesConfigUrls = ConfigurationOption.stringsOption()
```

Comment:
I have just merged it (https://github.com/stagemonitor/stagemonitor/pull/354) but it would be great if you could use `ConfigurationOption.urlsOption()`

Reply:
I can't find a ConfigurationOption.urlsOption() builder. What am I missing?

Reply:
You'll probably have to merge in the changes from master

Reply:
Haha, I already did so at 11:08 and you merged your feature branch at 11:09 ;-)